### PR TITLE
Popup details

### PR DESCRIPTION
**Pull request by sevinchek**
In this branch we:

- Created the layout of the popup window
- Created the event of the popup window
- Added projects to the object array
- Made the HTML Projects Section with Javascript

### CODE REVIEW

File: scripts/menu.js [hunk 1]
```diff
@@ -1,9 +1,267 @@
...
+  if (i > 0 && i < 7) {
+    // Create HTML Skeleton for 6 similar projects
+    const liCards = document.createElement('li');
+    liCards.className = `prof-art prof-art-${i} blur`;
+    ulContainer.appendChild(liCards);
+    let h4li = document.createElement('h4');
+    h4li.className = 'mob-h4';
+    liCards.appendChild(h4li);
+    h4li = document.createElement('h4');
+    h4li.className = 'desk-h4';
```
Comment (MahmoudBakr23) [inline scripts/menu.js:131]: Please notice that you're building the logic for adding projects to your portfolio in the near future easily! So, it doesn't make sense if you're making 6 similar elements in the projects' array right? Please tell me what you think!
The point is to dynamically (easily) add more and more projects to your portfolio with minimum effort (without changing too much code each time you want to add a new project) Kindly consider the following:- 

- [ ] Kindly remove the repeated code in the projects' array and apply the following:
1- Each project's pop-up should have its own image. Currently, all the modals have the same image!
2-

File: scripts/menu.js [hunk 2]
```diff
@@ -1,9 +1,267 @@
...
+    liTech = document.createElement('li');
+    liTech.className = 'bootstrap';
+    ulTech.appendChild(liTech);
+    liTech = document.createElement('li');
+    liTech.className = 'ruby';
+    ulTech.appendChild(liTech);
+    const liBtn = document.createElement('button');
+    liBtn.className = 'orng-btn popup-btn';
+    liBtn.innerHTML = 'See Project';
+    liCards.appendChild(liBtn);
```
Comment (MahmoudBakr23) [inline scripts/menu.js:150]: ### **OPTIONAL**

- [ ] On desktop screen size. All the projects should have the `See Project` button and each one opens a modal that belongs to it regarding the returned project's (object) data from the array you created.

_Also, please see the image down below for more clarification._
Screenshot1

**Review by MahmoudBakr23 — APPROVED** [review]
Hello @sevinchek and @alonsofl 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Linter is correct 💯 
- Descriptive PR headline, commit messages, and summary 💯 
- Good use of JS array's objects and DOM manipulation 💯
- Well done following the template's design flow 💯 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] It'd be great to update the live demo link in the README file with the latest changes.

_Please check the comments under the review._

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by sevinchek** [comment]
Thanks, @MahmoudBakr23. I'll work on your recommendations outside the program time.

**Pull request merged**